src/world.rs

```diff
@@ -73,6 +74,18 @@ impl World {
         self.dns.lookup(host)
     }
 
+    pub(crate) fn hold(&mut self, a: SocketAddr, b: SocketAddr) {
+        self.topology.hold(a, b);
+    }
+
+    // TODO: Should all held packets be immediately released, or should they be
```

Comment:
I'll speculate and say that I don't think they need to flow through all the existing turmoil machinery. We've got two paradigms here: run some longer tests with seeded random on and run shorter tests with it off and full control (ie hold a link).

I've been fighting this a bit with streams... obviously things like message re-order and drop are impossible in a TCP. We really need to do things at a higher level for that to work, ie re-order based on distinct connections.

I could see a case here though where you "hold", then a network partition happens, then "release" and...?